src/components/EntryPortal.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import * as tools from "../functions";
 import * as hospital_index from "../data/hospital_index";
 import HospitalResult from "./HospitalResult";
-import { withRouter } from "react-router-dom";
+import { withRouter, Link } from "react-router-dom";
```

Comment:
From here on down is a bunch of front-end changes?

Reply:
I didn't make any changes here, so I think it might have just been lagging behind commits they made in master earlier. Same with everything else that isn't in the functions folder.